contracts/its-stacks-translator/src/contract/query.rs

```diff
@@ -0,0 +1,18 @@
+use cosmwasm_std::{Binary, HexBinary};
+use interchain_token_service_std::HubMessage;
+
+use crate::contract::its_clarity_bytes_to_hub_message::its_clarity_bytes_to_hub_message;
+use crate::contract::its_hub_message_to_clarity_bytes::its_hub_message_to_clarity_bytes;
+use crate::error::ContractError;
+
+pub fn clarity_bytes_to_hub_message(clarity_payload: HexBinary) -> Result<Binary, ContractError> {
```

Comment:
```suggestion
pub fn bytes_to_hub_message(payload: HexBinary) -> Result<Binary, ContractError> {
```
clarity is implied by the contract name

Reply:
Done